contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseAppendOutputOperator.java

```diff
@@ -57,10 +58,10 @@ public AbstractHBaseAppendOutputOperator() {
   }
```

Comment:
store is not used any more in the new version. Is it necessary to create it here? If a subclass needs it, it can create it, right?